README.md

```diff
@@ -26,3 +26,32 @@ app.run();
 ```
 
 More complex examples and documentation are coming.
+
+## CSP
+
+`config.ts`
+
+```typescript
+import type {AppConfig} from '@gravity-ui/nodekit';
+import {csp} from '@gravity-ui/expresskit';
+
+const config: Partial<AppConfig> = {
+    // cspDisable: true,
```

Comment:
Why not disable by default?

Reply:
Yeah, let's change default to off?

> replace cspDisable with cspEnable, if we don't want to force users to use it.

Looks good